Extension/package.json

```diff
@@ -456,6 +456,16 @@
           "default": null,
           "description": "The value to use for the system include path. If set, it overrides the system include path acquired via \"compilerPath\" and \"compileCommands\" settings.",
           "scope": "resource"
+        },
+        "C_Cpp.updateChannel": {
+          "type": "string",
+          "enum": [
+            "Default",
+            "Insiders"
+          ],
+          "default": "Default",
```

Comment:
Can you add code to set the default to "Insiders" if they're using VS Code Insiders or an insiders vsix? There is related code in finalizeExtensionActivation().

Reply:
Absolutely! @bobbrow and I briefly discussed that and it should definitely be a part of the change.